contracts/cw721-expiration/examples/schema.rs

```diff
@@ -1,10 +1,10 @@
 use cosmwasm_schema::write_api;
-use cw721_expiration::msg::{ExecuteMsg, InstantiateMsg, QueryMsg};
+use cosmwasm_std::Empty;
+use cw721_expiration::msg::{InstantiateMsg, QueryMsg};
 
 fn main() {
     write_api! {
         instantiate: InstantiateMsg,
-        execute: ExecuteMsg,
```

Comment:
Nit, should include execute messages in schemas because `ts-codegen` needs them for frontend tooling.

Reply:
Thx! Missed that. Fixed and added.